### PR TITLE
Update Jira Server documentation

### PR DESCRIPTION
**Pull request by alena-babrova**

### CODE REVIEW

File: src/md/src/GettingStarted/Integrations.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ Users can reinforce ReportPortal with adding additional integrtions with:
 
 If you want to integrate ReportPortal with these external systems, and you can not find a needed tab on the Project Settings, please check the section in documentation [Plugins](https://reportportal.io/docs/Plugins).
 
-Integration configurations can be added on the global level (for all projects on the instance) on the Management Board or the project level (only for one project) on Project Settings.
+Integration configurations can be added on the global level (for all projects on the instance) on the Administrate area or the project level (only for one project) on Project Settings.
 
 If you have another configuration than others project have or you want to integrate only your project with an external system, you should perform the next actions:
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:18]: ...than other projects or you want...

File: src/md/src/GettingStarted/Integrations.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ Users can reinforce ReportPortal with adding additional integrtions with:
...
 
 1. Login ReportPortal as PROJECT MANAGER or ADMIN
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:20]: Log in to

File: src/md/src/GettingStarted/Integrations.md [hunk 3]
```diff
@@ -38,38 +38,38 @@ In this case, your settings will be deleted, and integration will use global set
 
 You always can reset to the global settings.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:39]: can always

File: src/md/src/GettingStarted/Integrations.md [hunk 4]
```diff
@@ -38,38 +38,38 @@ In this case, your settings will be deleted, and integration will use global set
...
 
-## Jira Integration 
+## Jira Server Integration 
 
-For Jira plugin installation please check [documentation](https://reportportal.io/docs/Plugins%3Eupload-plugin)
+For Jira Server plugin installation please check [documentation](https://reportportal.io/docs/Plugins%3Eupload-plugin)
 
-Integration with a JIRA can be required for projects that collect defects in a separate tracking tool. The integration provides an exchange of information between ReportPortal and the JIRA, such as posting issues and linking issues, getting updates on their statuses. 
+Integration with a JIRA Server can be required for projects that collect defects in a separate tracking tool. The integration provides an exchange of information between ReportPortal and the JIRA Server, such as posting issues and linking issues, getting updates on their statuses.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:45]: the JIRA Server

File: src/md/src/GettingStarted/Integrations.md [hunk 5]
```diff
@@ -38,38 +38,38 @@ In this case, your settings will be deleted, and integration will use global set
...
-For Jira plugin installation please check [documentation](https://reportportal.io/docs/Plugins%3Eupload-plugin)
+For Jira Server plugin installation please check [documentation](https://reportportal.io/docs/Plugins%3Eupload-plugin)
 
-Integration with a JIRA can be required for projects that collect defects in a separate tracking tool. The integration provides an exchange of information between ReportPortal and the JIRA, such as posting issues and linking issues, getting updates on their statuses. 
+Integration with a JIRA Server can be required for projects that collect defects in a separate tracking tool. The integration provides an exchange of information between ReportPortal and the JIRA Server, such as posting issues and linking issues, getting updates on their statuses. 
 
 **Permission:** 
 
-The *Administrator* can upload a plugin and make global integrations on the Plugin tab on the Management board. Also, the Administrator can unlink project configurations from global. 
+The *Administrator* can upload a plugin and make global integrations on the Plugin tab in the Administrate area. Also, the Administrator can unlink project configurations from global.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:49]: create global integrations

File: src/md/src/GettingStarted/Integrations.md [hunk 6]
```diff
@@ -13,13 +13,13 @@ Users can reinforce ReportPortal with adding additional integrtions with:
 
 If you want to integrate ReportPortal with these external systems, and you can not find a needed tab on the Project Settings, please check the section in documentation [Plugins](https://reportportal.io/docs/Plugins).
 
-Integration configurations can be added on the global level (for all projects on the instance) on the Management Board or the project level (only for one project) on Project Settings.
+Integration configurations can be added on the global level (for all projects on the instance) on the Administrate area or the project level (only for one project) on Project Settings.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:16]: может лучше "in the Administrate section", надо подумать, Administrate area - это административная зона

File: src/md/src/GettingStarted/Integrations.md [hunk 7]
```diff
@@ -38,38 +38,38 @@ In this case, your settings will be deleted, and integration will use global set
...
+#### Add new global Jira Server integration
 
 If all projects on your instance are using the same Jira projects, you will be able to configure **global integrations** which will be applied on all projects. 
 
 1. Login as ADMIN
 
 2. Go to Administrative > Plugins
 
-3. Click on JIRA plugin pannel
+3. Click on JIRA Server plugin pannel
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:63]: panel

File: src/md/src/GettingStarted/Integrations.md [hunk 8]
```diff
@@ -136,33 +136,31 @@ If you need to edit Jira integration authorization parameters, please perform ac
...
 
-If you need to edit fields which should be posted in Jira, please perform actions:
+If you need to edit fields which should be posted in Jira Server, please perform actions:
 
 1. Login as ADMIN
 
 2. Go to Administrative > Plugins
 
-3. Click on JIRA plugin pannel
+3. Click on JIRA Server plugin pannel
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:145]: panel

File: src/md/src/GettingStarted/Integrations.md [hunk 9]
```diff
@@ -123,11 +123,11 @@ If you need to edit Jira integration authorization parameters, please perform ac
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:123]: panel- line 122

File: src/md/src/GettingStarted/Integrations.md [hunk 10]
```diff
@@ -136,33 +136,31 @@ If you need to edit Jira integration authorization parameters, please perform ac
...
 
-3. Click on JIRA plugin pannel
+3. Click on JIRA Server plugin pannel
 
 4. Click on a tab with existing integration
 
-5. Click on the button "Configure"
+5. Click on "Configure" button
 
 6. Choose issue type from a drop-down
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:151]: the drop-down

File: src/md/src/GettingStarted/Integrations.md [hunk 11]
```diff
@@ -172,35 +170,35 @@ If you need to remove Jira integration, please perform actions:
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:170]: panel - line 169

File: src/md/src/GettingStarted/Integrations.md [hunk 12]
```diff
@@ -172,35 +170,35 @@ If you need to remove Jira integration, please perform actions:
...
+If each project needs different Jira Server configurations, you need to unlink a project from Global configurations and add a project configuration. It means that now when new global integration will be added to the ReportPortal, it would not apply to the unlinked project. 
 
 For that,
 
 1. Login as an ADMIN or Project Manager
 
 2. Go to Project Settings > Integrations
 
-3. Click on the JIRA integration pannel
+3. Click on the JIRA Server integration pannel
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:192]: panel

File: src/md/src/GettingStarted/Integrations.md [hunk 13]
```diff
@@ -172,35 +170,35 @@ If you need to remove Jira integration, please perform actions:
...
 
 
-### Project JIRA integration
+### Project JIRA Server integration
 
-#### Add new project Jira integration
+#### Add new project Jira Server integration
 
-If each project needs different Jira configurations, you need to unlink a project from Global configurations and add a project configuration. It means that now when new global integration will be added to the ReportPortal, it would not apply to the unlinked project. 
+If each project needs different Jira Server configurations, you need to unlink a project from Global configurations and add a project configuration. It means that now when new global integration will be added to the ReportPortal, it would not apply to the unlinked project.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:184]: If any project...
...you should unlink...
...when a new global integration is added... it won't be applied to

File: src/md/src/GettingStarted/Integrations.md [hunk 14]
```diff
@@ -212,32 +210,32 @@ This feature gives you the ability to choose which type you will post a ticket.
 
 For choosing needed issue type and predefined field for the chosen issue, you should perform the following actions:
 
-1. Click button "Configure"
+1. Click on "Configure" button
 
 2. Choose issue type from a drop-down
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:215]: the drop-down

File: src/md/src/GettingStarted/Integrations.md [hunk 15]
```diff
@@ -212,32 +210,32 @@ This feature gives you the ability to choose which type you will post a ticket.
...
 
 3. Check needed fields and fill them if you need
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:217]: if needed

File: src/md/src/GettingStarted/Integrations.md [hunk 16]
```diff
@@ -81,27 +81,27 @@ This feature gives you the ability to choose which type you will post a ticket.
...
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:88]: if needed

File: src/md/src/GettingStarted/Integrations.md [hunk 17]
```diff
@@ -136,33 +136,31 @@ If you need to edit Jira integration authorization parameters, please perform ac
...
+3. Click on JIRA Server plugin pannel
 
 4. Click on a tab with existing integration
 
-5. Click on the button "Configure"
+5. Click on "Configure" button
 
 6. Choose issue type from a drop-down
 
 7. Check needed fields and fill them if you need
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:153]: if needed

File: src/md/src/GettingStarted/Integrations.md [hunk 18]
```diff
@@ -368,13 +364,13 @@ This feature gives you the ability to choose which type you will post a ticket.
 
 For choosing needed issue type and predefined field for the chosen issue, you should perform the following actions:
 
-1. Click button "Configure"
+1. Click on "Configure" button
 
 2. Choose issue type from a drop-down
 
 3. Check needed fields and fill them if you need
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:371]: if needed

File: src/md/src/GettingStarted/Integrations.md [hunk 19]
```diff
@@ -430,16 +426,13 @@ If you need to edit fields which should be posted in RALLY, please perform actio
 
 4. Click on a tab with existing integration
 
-5. Click on the button "Configure"
+5. Click on "Configure" button
 
 6. Choose issue type from a drop-down
 
 7. Check needed fields and fill them if you need
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:433]: if needed

File: src/md/src/GettingStarted/Integrations.md [hunk 20]
```diff
@@ -491,18 +484,18 @@ This feature gives you the ability to choose which type you will post a ticket w
 
 For choosing needed issue type and predefined field for the chosen issue, you should perform the following actions:
 
-1. Click button "Configure"
+1. Click on "Configure" button
 
 2. Choose issue type from a drop-down
 
 3. Check needed fields and fill them if you need
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:491]: Fill in the fields if needed

File: src/md/src/GettingStarted/Integrations.md [hunk 21]
```diff
@@ -212,32 +210,32 @@ This feature gives you the ability to choose which type you will post a ticket.
...
 
-If you want to delete project integrations with Jira and link your project with global configurations, please perform actions described below:
+If you want to delete project integrations with Jira Server and link your project with global configurations, please perform actions described below:
 
 1. Login as an ADMIN or Project Manager
 
 2. Go to Project Settings > Integrations
 
-3. Click on the JIRA integration pannel
+3. Click on the JIRA Server integration pannel
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:236]: panel

File: src/md/src/GettingStarted/Integrations.md [hunk 22]
```diff
@@ -350,14 +346,14 @@ If all projects on your instance are using the same RALLY  projects, you will be
 
 3. Click on RALLY plugin pannel
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:347]: panel

File: src/md/src/GettingStarted/Integrations.md [hunk 23]
```diff
@@ -472,14 +465,14 @@ For that,
 
 3. Click on the RALLY integration pannel
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:466]: panel

File: src/md/src/GettingStarted/Integrations.md [hunk 24]
```diff
@@ -514,7 +507,7 @@ If you want to delete project integrations with RALLY and link your project with
...
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:508]: panel

File: src/md/src/GettingStarted/Integrations.md [hunk 25]
```diff
@@ -212,32 +210,32 @@ This feature gives you the ability to choose which type you will post a ticket.
 
 For choosing needed issue type and predefined field for the chosen issue, you should perform the following actions:
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:211]: To choose a needed...

File: src/md/src/GettingStarted/Integrations.md [hunk 26]
```diff
@@ -212,32 +210,32 @@ This feature gives you the ability to choose which type you will post a ticket.
...
 3. Check needed fields and fill them if you need
 
-4. Click on "Submit"
+4. Click on "Submit" button
 
 
 Now team members on **this project** will be able to submit issues in Jira. Options for Post Issue / Link issue are activated.
 
-If you needed you can add more integrations, by clicking on "Add integration".
+If you needed you can add more integrations, by clicking on "Add integration" button.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:224]: You can add more... без "if you need" и без запятой

File: src/md/src/GettingStarted/Integrations.md [hunk 27]
```diff
@@ -491,18 +484,18 @@ This feature gives you the ability to choose which type you will post a ticket w
...
 3. Check needed fields and fill them if you need
 
-4. Click on "Submit"
+4. Click on "Submit" button
 
 
 Now team members on **this project** will be able to submit issues in RALLY. Options for Post Issue / Link issue are activated.
 
-If you needed you can add more integrations, by clicking on "Add integration".
+If you needed you can add more integrations, by clicking on "Add integration" button.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:498]: You can add more... без "if you need" и без запятой

File: src/md/src/GettingStarted/Integrations.md [hunk 28]
```diff
@@ -246,27 +244,27 @@ If you want to delete project integrations with Jira and link your project with
 
 ### Some tricks when you create a new connection:
 
-1. Verify that the link to the JIRA system is correct. There are some variants are possible, for instance: https://jira.company.com/jira https://jiraeu.company.com
+1. Verify that the link to the JIRA Server system is correct. There are some variants are possible, for instance: https://jira.company.com/jira https://jiraeu.company.com
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:247]: There are several possible variants

File: src/md/src/GettingStarted/Integrations.md [hunk 29]
```diff
@@ -246,27 +244,27 @@ If you want to delete project integrations with Jira and link your project with
...
 
-3. Verify the username and password data. Make sure, that the login name and not the email are in the username field. In case all the data above is correct, but the error appears again, check whether the user's credentials to JIRA are not expired. As far as JIRA sends the request in HTML format, we are not able to display the real reason for the error. To check and/or resolve the issue, please do the next steps:
+3. Verify the username and password data. Make sure, that the login name and not the email are in the username field. In case all the data above is correct, but the error appears again, check whether the user's credentials to JIRA Server are not expired. As far as JIRA Server sends the request in HTML format, we are not able to display the real reason for the error. To check and/or resolve the issue, please do the next steps:
 
-   *    Open JIRA page
-   *    Login JIRA with domain credentials using basic authorization (i.e., fill in user's login name and password into the fields)
+   *    Open JIRA Server page
+   *    Login JIRA Server with domain credentials using basic authorization (i.e., fill in user's login name and password into the fields)
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:254]: Log in to

File: src/md/src/GettingStarted/Integrations.md [hunk 30]
```diff
@@ -38,38 +38,38 @@ In this case, your settings will be deleted, and integration will use global set
...
 
-### Global JIRA integration
+### Global JIRA Server integration
 
-#### Add new global Jira integration
+#### Add new global Jira Server integration
 
 If all projects on your instance are using the same Jira projects, you will be able to configure **global integrations** which will be applied on all projects. 
 
 1. Login as ADMIN
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:59]: Log in

File: src/md/src/GettingStarted/Integrations.md [hunk 31]
```diff
@@ -111,9 +111,9 @@ User can add several integrations, but with a different name to the **one JIRA p
 > [ ![UseCaseWithTwoIntegrations](Images/userGuide/gettingStarted/JiraProject.png) ](https://youtu.be/med4tuL4plU)
 
 
-#### Update global Jira integration
+#### Update global Jira Server integration
 
-If you need to edit Jira integration authorization parameters, please perform actions:
+If you need to edit Jira Server integration authorization parameters, please perform actions:
 
 1. Login as ADMIN
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:118]: Log in

File: src/md/src/GettingStarted/Integrations.md [hunk 32]
```diff
@@ -136,33 +136,31 @@ If you need to edit Jira integration authorization parameters, please perform ac
 > You can edit the "Integration name" only. If you need other changes, please submit new integration.
 
 
-If you need to edit fields which should be posted in Jira, please perform actions:
+If you need to edit fields which should be posted in Jira Server, please perform actions:
 
 1. Login as ADMIN
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:141]: Log in

File: src/md/src/GettingStarted/Integrations.md [hunk 33]
```diff
@@ -136,33 +136,31 @@ If you need to edit Jira integration authorization parameters, please perform ac
...
 [ ![Edit Global Jira Integration](Images/userGuide/gettingStarted/JiraGlobal.png) ](https://youtu.be/yvYwxs4zTYA)
 
 
-#### Remove global Jira integration
+#### Remove global Jira Server integration
 
-If you need to remove Jira integration, please perform actions:
+If you need to remove Jira Server integration, please perform actions:
 
 1. Login as ADMIN
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:165]: Log in

File: src/md/src/GettingStarted/Integrations.md [hunk 34]
```diff
@@ -172,35 +170,35 @@ If you need to remove Jira integration, please perform actions:
...
 
-#### Add new project Jira integration
+#### Add new project Jira Server integration
 
-If each project needs different Jira configurations, you need to unlink a project from Global configurations and add a project configuration. It means that now when new global integration will be added to the ReportPortal, it would not apply to the unlinked project. 
+If each project needs different Jira Server configurations, you need to unlink a project from Global configurations and add a project configuration. It means that now when new global integration will be added to the ReportPortal, it would not apply to the unlinked project. 
 
 For that,
 
 1. Login as an ADMIN or Project Manager
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:188]: Log in

File: src/md/src/GettingStarted/Integrations.md [hunk 35]
```diff
@@ -212,32 +210,32 @@ This feature gives you the ability to choose which type you will post a ticket.
...
 
 [ ![ProjectJiraIntegration](Images/userGuide/gettingStarted/JiraProject.png) ](https://youtu.be/tCRh30KDpKc)
 
-#### Reset to project Jira Integrations
+#### Reset to project Jira Server Integrations
 
-If you want to delete project integrations with Jira and link your project with global configurations, please perform actions described below:
+If you want to delete project integrations with Jira Server and link your project with global configurations, please perform actions described below:
 
 1. Login as an ADMIN or Project Manager
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:232]: Log in

File: src/md/src/GettingStarted/Integrations.md [hunk 36]
```diff
@@ -246,27 +244,27 @@ If you want to delete project integrations with Jira and link your project with
...
+   *    Now try to establish the connection to JIRA Server on the ReportPortal project.
 
 4. SSL instance of JIRA (even cloud version) can be accessed by JIRA API token, used instead of a password.
 After you have connected Jira and ReportPortal, you can choose an issue type that you will be able to add to Jira during the “Post Issue” operation.  Also, the user can add predefined fields that the user can fill.
 
-### Post issue to Jira
+### Post issue to Jira Server
 
-By posting the issue in Jira, we mean a possibility to create a new issue in Jira from ReportPortal and upload logs and attachments from an execution.
+By posting the issue in Jira Server, we mean a possibility to create a new issue in Jira from ReportPortal and upload logs and attachments from an execution.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:265]: Posting an issue to Jira Server means to create

File: src/md/src/GettingStarted/Integrations.md [hunk 37]
```diff
@@ -278,19 +276,19 @@ Post issue via Step view
 
 3. Choose needed item
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:277]: a needed

File: src/md/src/GettingStarted/Integrations.md [hunk 38]
```diff
@@ -278,19 +276,19 @@ Post issue via Step view
...
+4. Click on "Actions" button
 
-5. Choose an action "Post issue"
+5. Choose "Post issue" option
 
 6. Fill a posting form with valid data
 
 ```javascript
-`BTS`: if you have configured Jira and Rally integrations, you will be able to choose needed
+`BTS`: if you have configured BTS integrations, you will be able to choose needed
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:286]: to choose between them

File: src/md/src/GettingStarted/Integrations.md [hunk 39]
```diff
@@ -300,30 +298,28 @@ Post issue via Step view
 9. A label with issue ID will be added to the test item
 
 
+### Link an issue with an issue in Jira Server
 
-
-### Link an issue with an issue in Jira
-
-By linking issue with an issue in Jira, we mean a possibility to a clickable link to an existing issue in Jira from ReportPortal that will show a status of this issue. 
+By linking issue with an issue in Jira Server, we mean a possibility to a clickable link to an existing issue in Jira from ReportPortal that will show a status of this issue.
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:303]: Linking an issue with an issue in Jira Server means adding a clickable link

File: src/md/src/GettingStarted/Integrations.md [hunk 40]
```diff
@@ -300,30 +298,28 @@ Post issue via Step view
...
-
-By linking issue with an issue in Jira, we mean a possibility to a clickable link to an existing issue in Jira from ReportPortal that will show a status of this issue. 
+By linking issue with an issue in Jira Server, we mean a possibility to a clickable link to an existing issue in Jira from ReportPortal that will show a status of this issue. 
 
 [ ![Link issue](Images/userGuide/gettingStarted/Link issue.png) ](Images/userGuide/gettingStarted/Link issue.png)
 
-Post issue via Step view
+Link issue via Step view
 
 1. Login ReportPortal as Admin, PM, Member, Customer or Operator
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:309]: Log in to

File: src/md/src/GettingStarted/Integrations.md [hunk 41]
```diff
@@ -300,30 +298,28 @@ Post issue via Step view
...
 
 2. Come to a step view
 
 3. Choose needed item
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:313]: a needed

File: src/md/src/GettingStarted/Integrations.md [hunk 42]
```diff
@@ -300,30 +298,28 @@ Post issue via Step view
...
+By linking issue with an issue in Jira Server, we mean a possibility to a clickable link to an existing issue in Jira from ReportPortal that will show a status of this issue. 
 
 [ ![Link issue](Images/userGuide/gettingStarted/Link issue.png) ](Images/userGuide/gettingStarted/Link issue.png)
 
-Post issue via Step view
+Link issue via Step view
 
 1. Login ReportPortal as Admin, PM, Member, Customer or Operator
 
 2. Come to a step view
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:311]: Go to the step view

File: src/md/src/GettingStarted/Integrations.md [hunk 43]
```diff
@@ -300,30 +298,28 @@ Post issue via Step view
...
 
 3. Choose needed item
 
-4. Click on a button "Actions"
+4. Click on "Actions" button
 
-5. Choose an action "Link issue"
+5. Choose "Link issue" option
 
 6. Fill a posting form with valid data
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:319]: Fill in the "Post issue" form with...

File: src/md/src/GettingStarted/Integrations.md [hunk 44]
```diff
@@ -300,30 +298,28 @@ Post issue via Step view
...
 
 ```javascript
-`BTS`: if you have configured Jira and Rally integrations, you will be able to choose needed
+`BTS`: if you have configured BTS integrations, you will be able to choose needed
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:322]: ... be able to choose between them

File: src/md/src/GettingStarted/Integrations.md [hunk 45]
```diff
@@ -81,27 +81,27 @@ This feature gives you the ability to choose which type you will post a ticket.
 
 For choosing needed issue type and predefined field for the chosen issue, you should perform the following actions:
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:82]: To choose a needed...

File: src/md/src/GettingStarted/Integrations.md [hunk 46]
```diff
@@ -368,13 +364,13 @@ This feature gives you the ability to choose which type you will post a ticket.
...
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:365]: To choose a needed...

File: src/md/src/GettingStarted/Integrations.md [hunk 47]
```diff
@@ -491,18 +484,18 @@ This feature gives you the ability to choose which type you will post a ticket w
...
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:485]: To choose a needed...

File: src/md/src/GettingStarted/Integrations.md [hunk 48]
```diff
@@ -368,13 +364,13 @@ This feature gives you the ability to choose which type you will post a ticket.
...
 
-1. Click button "Configure"
+1. Click on "Configure" button
 
 2. Choose issue type from a drop-down
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:369]: the drop-down

File: src/md/src/GettingStarted/Integrations.md [hunk 49]
```diff
@@ -430,16 +426,13 @@ If you need to edit fields which should be posted in RALLY, please perform actio
 
 4. Click on a tab with existing integration
 
-5. Click on the button "Configure"
+5. Click on "Configure" button
 
 6. Choose issue type from a drop-down
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:431]: the drop-down

File: src/md/src/GettingStarted/Integrations.md [hunk 50]
```diff
@@ -81,27 +81,27 @@ This feature gives you the ability to choose which type you will post a ticket.
 
 For choosing needed issue type and predefined field for the chosen issue, you should perform the following actions:
 
-1. Click button "Configure"
+1. Click on "Configure" button
 
 2. Choose issue type from a drop-down
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:86]: the drop-down

File: src/md/src/GettingStarted/Integrations.md [hunk 51]
```diff
@@ -491,18 +484,18 @@ This feature gives you the ability to choose which type you will post a ticket w
...
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:489]: the drop-down

File: src/md/src/GettingStarted/Integrations.md [hunk 52]
```diff
@@ -472,14 +465,14 @@ For that,
 
 3. Click on the RALLY integration pannel
 
-4. Click on "Unlink and setup manually"
+4. Click on "Unlink and setup manually" button
 
 5. Fill and confirm authorization form
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:470]: the authorization form

File: src/md/src/GettingStarted/Integrations.md [hunk 53]
```diff
@@ -38,38 +38,38 @@ In this case, your settings will be deleted, and integration will use global set
...
 
 2. Go to Administrative > Plugins
 
-3. Click on JIRA plugin pannel
+3. Click on JIRA Server plugin pannel
 
-4. Click on "Add integration"
+4. Click on "Add integration" button
 
 5. Fill and confirm authorization form
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:67]: the authorization form

File: src/md/src/GettingStarted/Integrations.md [hunk 54]
```diff
@@ -172,35 +170,35 @@ If you need to remove Jira integration, please perform actions:
...
 
 2. Go to Project Settings > Integrations
 
-3. Click on the JIRA integration pannel
+3. Click on the JIRA Server integration pannel
 
-4. Click on "Unlink and setup manually"
+4. Click on "Unlink and setup manually" button
 
 5. Fill and confirm authorization form
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:196]: the authorization form

File: src/md/src/GettingStarted/Integrations.md [hunk 55]
```diff
@@ -350,14 +346,14 @@ If all projects on your instance are using the same RALLY  projects, you will be
 
 3. Click on RALLY plugin pannel
 
-4. Click on "Add integration"
+4. Click on "Add integration" button
 
 5. Fill and confirm authorization form
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:351]: the authorization form

File: src/md/src/GettingStarted/Integrations.md [hunk 56]
```diff
@@ -514,7 +507,7 @@ If you want to delete project integrations with RALLY and link your project with
 
 3. Click on the RALLY integration pannel
 
-4. Click on "Reset to global settings"
+4. Click on "Reset to global settings" button
 
 5. Confirm actions
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:512]: Confirm the action

File: src/md/src/GettingStarted/Integrations.md [hunk 57]
```diff
@@ -212,32 +210,32 @@ This feature gives you the ability to choose which type you will post a ticket.
...
 
 2. Go to Project Settings > Integrations
 
-3. Click on the JIRA integration pannel
+3. Click on the JIRA Server integration pannel
 
-4. Click on "Reset to global settings"
+4. Click on "Reset to global settings" button
 
 5. Confirm actions
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:240]: Confirm the action

File: src/md/src/GettingStarted/Integrations.md [hunk 58]
```diff
@@ -532,9 +525,9 @@ Post issue via Step view
 
 3. Choose needed item
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:526]: a needed

File: src/md/src/GettingStarted/Integrations.md [hunk 59]
```diff
@@ -570,9 +563,9 @@ Post issue via Step view
...
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:564]: a needed

File: src/md/src/GettingStarted/Integrations.md [hunk 60]
```diff
@@ -278,19 +276,19 @@ Post issue via Step view
...
 
-4. Click on a button "Actions"
+4. Click on "Actions" button
 
-5. Choose an action "Post issue"
+5. Choose "Post issue" option
 
 6. Fill a posting form with valid data
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:283]: Fill in the "Post issue" form with...

File: src/md/src/GettingStarted/Integrations.md [hunk 61]
```diff
@@ -532,9 +525,9 @@ Post issue via Step view
...
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:532]: Fill in the "Post issue" form with...

File: src/md/src/GettingStarted/Integrations.md [hunk 62]
```diff
@@ -570,9 +563,9 @@ Post issue via Step view
 
 3. Choose needed item
 
-4. Click on a button "Actions"
+4. Click on "Actions" button
 
-5. Choose an action "Link issue"
+5. Choose "Link issue" option
 
 6. Fill a posting form with valid data
```
Comment (VolhaKarenko) [inline src/md/src/GettingStarted/Integrations.md:570]: Fill in the "Post issue" form with...

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**